tables/packetdescriptions.txt

```diff
@@ -413,7 +428,6 @@
 080B Booking Delete
 080E Party User HP Info
 080F Deal Item Added (Other)
-081B Buying Store Item Update
```

Comment:
this seems that u just removed and not added again

Reply:
this line is duplicated below